module/LineTraceArea.cpp

```diff
@@ -123,5 +123,23 @@ void LineTraceArea::runLineTraceAreaShortcut()
     controller.sleep();
   }
 
-  lineTracer.run(straightDistance3, targetBrightness, 100, PidGain(4, 1, 1));
+  //外れた時用
+  int count = 0;
+  initialDistance = Mileage::calculateMileage(measurer.getRightCount(), measurer.getLeftCount());
+  COLOR color = ColorJudge::getColor(measurer.getRawColor());
+  if(color == COLOR::WHITE) {
+    while(true) {
+      COLOR color = ColorJudge::getColor(measurer.getRawColor());
+      if(color == COLOR::BLACK) {
+        break;
+      }
+      controller.setRightMotorPwm(90);
+      controller.setLeftMotorPwm(80);
+      controller.sleep();
+    }
+  }
```

Comment:
これでいいんじゃない？あとここコメント欲しいかな
黒線に乗るまで左に旋回するとかかな？
```suggestion
    while(color != COLOR::WHITE) {
      controller.setRightMotorPwm(90);
      controller.setLeftMotorPwm(80);
      controller.sleep();
    }
```